ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -443,6 +448,11 @@ spec:
     - name: get-manifest-digests
       runAfter:
         - publish-bundle
+      when:
+        - input: *operatorDistribution
+          operator: notin
+          values:
+            - undistributed
```

Comment:
Just a suggestion: since we are reusing the same `when` clause all over the place, perhaps it would make sense to create an anchor for the entire input/operator/values block. WDYT?

Reply:
Good idea. Done.